ion-sfu-gstreamer-send/main.go

```diff
@@ -47,6 +49,22 @@ func main() {
 	}
 	// new sdk engine
 	e := sdk.NewEngine(config)
+       // e.RegisterCodec(webrtc.RTPCodecParameters{
+   //RTPCodecCapability: webrtc.RTPCodecCapability{MimeType: H264, ClockRate: 90000, Channels: 0, SDPFmtpLine: "packetization-mode=1;profile-level-id=42e01f", RTCPFeedback: nil},
+   //PayloadType:        webRTC.PayloadType(107),
+//}, webrtc.RTPCodecTypeVideo); 
+ 
+        // Create a MediaEngine object to configure the supported codec
+	m := &webrtc.MediaEngine{}
+
+	// Setup the codecs you want to use.
+	// We'll use a VP8 and Opus but you can also define your own
+	if err := m.RegisterCodec(webrtc.RTPCodecParameters{
```

Comment:
So you use `m` to register codec. But who uses `m` 
Client being constructed should use it somehow no ?

Reply:
Agreed. Not sure how to register a new codec using sdk new engine.
If you see line 52, i tried doing that but it didn't work late last night. let me try it again